src/main/java/org/commoncrawl/spark/CCIndex2Table.java

```diff
@@ -109,6 +110,10 @@ public static Row convertCdxLine(String line) {
 					cdx.crawl, cdx.subset);
 		} else {
 			Row h = cdx.uri.getHostName().asRow();
+			if( h.get(0) == null ) {
```

Comment:
Jason will re-test this as well - if it was a problem with real data, we should leave this in. If it's problem with testing/test data, we should probably remove this. (It's better to let the job fail - because we had bad data, etc)